README.md

```diff
@@ -1,101 +1,482 @@
-## loopback-connector-oracle
+# loopback-connector-oracle
 
-The Oracle Connector module for for [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler).
+[Oracle](https://www.oracle.com/database/index.html) is an object-relational database management system produced by Oracle Corporation. The `loopback-connector-oracle` module is the Oracle connector for the LoopBack framework.
 
-For full documentation, see the official [Documentation](http://loopback.io/doc/en/lb2/Oracle-connector.html).
+For more information, see the [LoopBack documentation](http://loopback.io/doc/en/lb3/Oracle-connector.html).
 
-**NOTE**: This connector is compatible with Node v.0.10.x, v.12.x, and iojs v1.6.2.  
+## Prerequisites
+
+**Node.js**: The Oracle connector is compatible with Node v.0.10.x, v.12.x, and later.  
+
+**MacOS**: The Oracle connector does not work on Mac OSX 10.11 (El Capitan) due to [SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection) runtime protections and restrictions.
+Specifically, DYLD environment variables are now ignored; hence, rendering the Oracle connector inoperable due to the DYLD_LIBRARY_PATH dependency.
+For a workaround, see [How to Disable SIP in OS X El Capitan](http://osxdaily.com/2015/10/05/disable-rootless-system-integrity-protection-mac-os-x/).
+
+**Windows**: On 64-bit Windows systems, the Oracle connector runs only on 64-bit version of Node.js.
+
+**Oracle**: The Oracle connector requires Oracle 8.x - 12.x.
```

Comment:
The node-oracledb add-on requires Oracle client libraries 11.2 onwards.  These can connect to Oracle DB 9.2 onwards.